src/features/onboarding/Onboarding/DaysBetweenInput.js

```diff
@@ -0,0 +1,56 @@
+import React, { useState, useEffect } from "react"
+import { TextField, InputAdornment } from "@material-ui/core"
+
+const minMax = {
+  min: 15,
+  max: 60,
+}
+
+const placeholder = "28"
+
+const DaysBetweenInput = ({ onChange, values, textFieldProps }) => {
+  const [value, setValue] = useState(values.daysBetween)
+
+  const onChangeDaysBetween = (e) => {
+    const currentValue = e.target.value.length
```

Comment:
I was thinking we juts brute force everything in here. 

If value is -1, return 27. 
If value is 1, return 29.
If > max, return max.
If < min, return min. 
Else return value.